core-plugins/src/e2e-test/features/joiner/JoinerMacros.feature

```diff
@@ -79,3 +82,63 @@ Feature:Joiner - Verification of Joiner pipeline with File as source and File as
     Then Close the pipeline logs
     Then Validate OUT record count of joiner is equal to IN record count of sink
     Then Validate output file generated by file sink plugin "fileSinkTargetBucket" is equal to expected output file "joinerMacroOutputFile"
+
+  @JOINER_TEST1 @JOINER_TEST2 @FILE_SINK_TEST
+  Scenario:To verify data is getting transferred from File to File successfully using Joiner plugin with additional macro arguments
```

Comment:
`To verify data is getting transferred from File to File successfully using Joiner plugin with advanced join with macro arguments`

similar the above `scenario` will be 

`To verify data is getting transferred from File to File successfully using Joiner plugin with basic join with macro arguments`

Reply:
done